lib/rack/attack.rb

```diff
@@ -53,6 +53,19 @@ def clear!
         @configuration.clear_configuration
       end
 
+      def reset!
+        store = cache.store
+
+        if store.respond_to?(:delete_matched)
+          store.delete_matched("#{cache.prefix}*")
+        else
+          raise(
+            MisconfiguredStoreError,
+            "Configured store #{store.class.name} doesn't respond to #delete_matched method"
```

Comment:
`MisconfiguredStoreError` was conceived to let a user know it need to reconfigure rack-attack or nothing will work.

In this particular case, not sure we're exactly there. This would more like "Everything will work fine with your current store but you won't be able to use the `#reset!` method"? Maybe `IncompatibleStoreError` for the constant?